### DISCUSSION TITLE
I get the following message when cypress try redirecting after of login

### DISCUSSION THREAD

**Discussion by abrilmaestre** [cypress-io/cypress#25541] [Questions and Help]
I do login through an subdomain from the same domain that baseURL but when cypress type credentials and click on Login button, it tries to redirects to home page of the application but I get the following in the test runner of cypress:

![2023-01-23_11-20](https://user-images.githubusercontent.com/83642384/214018698-6d36c3fe-8434-46ce-ac45-ed447b615c97.png)

I have tried the same scenario manually and it works correctly.

**Comment by portyu9** [comment]
Hi @abrilmaestre 

The screenshot indicates that the login redirect is **breaking out of Cypress's application iframe** and navigating the Cypress runner itself. That is why you see Cypress's `"You seem to have gotten lost"` page.

Enable Cypress's frame-busting protection:

```js
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  modifyObstructiveCode: true,
})
```

Since your login occurs on a different subdomain/origin, with current Cypress also run the login commands inside `cy.origin()`:

```js
cy.origin('https://login.example.com', () => {
  cy.get('[name="username"]').type('user')
  cy.get('[name="password"]').type('password')
  cy.get('button').click()
})
```

If the authentication origin itself contains frame-busting code, also enable:

```js
experimentalModifyObstructiveThirdPartyCode: true
```

So the fix is **prevent the authentication page from escaping Cypress's iframe and use `cy.origin()` for the login subdomain**.

**Reply by portyu9** [reply]
@abrilmaestre Please mark this answer as accepted if it helped, thank you.
